### PR TITLE
(Fix) [FtpStream] Missing "first" file in directory

### PR DESCRIPTION
**Pull request by DerDu**
Signed-off-by: Gerd Christian Kunze <gerdchristian.kunze@googlemail.com>

### CODE REVIEW

File: src/Wrappers/FtpStream.php [hunk 1]
```diff
@@ -229,9 +229,15 @@ public function dir_opendir($url, $options) {
 	}
 
 	public function dir_readdir() {
-		if (($filename = next($this->dir_list)) === false) {
-			return false;
-		}
+        if( $this->dir_pos === 0 ) {
```
Comment (emersion) [inline src/Wrappers/FtpStream.php]: Style: please use tabs, put spaces after `if`, don't put spaces inside parentheses

Comment (DerDu) [reply src/Wrappers/FtpStream.php]: ah ok.. we use other conventions ;)

**Comment by emersion** [comment]
Thanks!

**Comment by DerDu** [comment]
Welcome! :)

**Pull request merged**